template/lib.typ

```diff
@@ -166,12 +166,12 @@ text)
       // Choose between normal and emph text
       let body = if item.numbering == none {
           emph(item.body)
-
       } else {
         item.body
       }
 
       // Page assignation
+      //let page_numbering  = counter(page).get().first()
```

Comment:
Leftover comment?

Reply:
hahaha yes, it is

Reply:
I was triying diferent modifications to get the correct headers